lib/ReactViews/DataCatalog/DataCatalogGroup.jsx

```diff
@@ -61,6 +59,15 @@ const DataCatalogGroup = createReactClass({
         this.props.viewState.viewCatalogMember(this.props.group);
     },
 
+    removeUserAddedGroup() {
+      const parent = this.props.group.parent;
+      // can remove if not root group
+      if(parent && parent.parent) {
```

Comment:
Do we need the `parent.parent` check? I don't think it'll ever be false currently, and if this function were used differently in the future it might actually get in the way.

Actually, given the name, something like this would be pretty sensible:

```
const itemIndex = this.props.terria.catalog.userAddedDataGroup.items.indexOf(this.props.group);
if (itemIndex >= 0) {
    this.props.terria.catalog.userAddedDataGroup.items.splice(itemIndex, 1);
}
```

Reply:
yup~ put it in a new file